appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -185,10 +190,27 @@ export class CaseDetailsPage extends Page {
 		this.elements.costDecisionStatus().contains(text);
 	}
 
+	clickRemoveFileUpload(buttonIndex) {
```

Comment:
For this could you do something like  clickRemoveFileUpload(fileName) {
this.elements.removeFileUpload(fileName).click(); 
and then call out the file name in the test?

Reply:
You're right I could  - I've fixed this now to work based on the filename